[WARNING] ME(368760:281473515538736,MainProcess):2025-08-28-16:19:07.205.294 [mindspore/run_check/_check_version.py:412] Can not find the tbe operator implementation(need by mindspore-ascend). Please check whether the Environment Variable PYTHONPATH is set. For details, refer to the installation guidelines: https://www.mindspore.cn/install
/home/ma-user/anaconda3/envs/MindSpore/lib/python3.9/site-packages/numpy/core/getlimits.py:500: UserWarning: The value of the smallest subnormal for <class 'numpy.float64'> type is zero.
  setattr(self, word, getattr(machar, word).flat[0])
/home/ma-user/anaconda3/envs/MindSpore/lib/python3.9/site-packages/numpy/core/getlimits.py:89: UserWarning: The value of the smallest subnormal for <class 'numpy.float64'> type is zero.
  return self._float_to_str(self.smallest_subnormal)
/home/ma-user/anaconda3/envs/MindSpore/lib/python3.9/site-packages/numpy/core/getlimits.py:500: UserWarning: The value of the smallest subnormal for <class 'numpy.float32'> type is zero.
  setattr(self, word, getattr(machar, word).flat[0])
/home/ma-user/anaconda3/envs/MindSpore/lib/python3.9/site-packages/numpy/core/getlimits.py:89: UserWarning: The value of the smallest subnormal for <class 'numpy.float32'> type is zero.
  return self._float_to_str(self.smallest_subnormal)
[WARNING] ME(368760:281473515538736,MainProcess):2025-08-28-16:19:11.907.295 [mindspore/run_check/_check_version.py:412] Can not find the tbe operator implementation(need by mindspore-ascend). Please check whether the Environment Variable PYTHONPATH is set. For details, refer to the installation guidelines: https://www.mindspore.cn/install
[WARNING] DISTRIBUTED(368760,ffffa8e84930,python):2025-08-28-16:19:11.909.260 [mindspore/ccsrc/distributed/rpc/tcp/tcp_comm.cc:485] Connect] Connection 17 source: 127.0.0.1:38524, destination: 127.0.0.1:8118
[WARNING] DISTRIBUTED(368760,ffff3172b1e0,python):2025-08-28-16:19:11.909.274 [mindspore/ccsrc/distributed/rpc/tcp/tcp_comm.cc:79] ConnectedEventHandler] Connection from 127.0.0.1:38524 to 127.0.0.1:8118 is successfully created. System errno: Success
[WARNING] DISTRIBUTED(368760,ffffa8e84930,python):2025-08-28-16:19:11.909.304 [mindspore/ccsrc/distributed/rpc/tcp/tcp_comm.cc:494] Connect] Waiting for the state of the connection to 127.0.0.1:8118 to be connected...Retry number: 1
[WARNING] DISTRIBUTED(368760,ffffa8e84930,python):2025-08-28-16:19:12.409.491 [mindspore/ccsrc/distributed/rpc/tcp/tcp_comm.cc:485] Connect] Connection 18 source: 127.0.0.1:38534, destination: 127.0.0.1:8118
[WARNING] DISTRIBUTED(368760,ffffa8e84930,python):2025-08-28-16:19:12.409.517 [mindspore/ccsrc/distributed/rpc/tcp/tcp_comm.cc:494] Connect] Waiting for the state of the connection to 127.0.0.1:8118 to be connected...Retry number: 2
[WARNING] DISTRIBUTED(368760,ffff3272d1e0,python):2025-08-28-16:19:12.409.526 [mindspore/ccsrc/distributed/rpc/tcp/tcp_comm.cc:79] ConnectedEventHandler] Connection from 127.0.0.1:38534 to 127.0.0.1:8118 is successfully created. System errno: Success
[WARNING] DISTRIBUTED(368760,ffffa8e84930,python):2025-08-28-16:19:12.910.541 [mindspore/ccsrc/distributed/cluster/cluster_context.cc:246] BuildCluster] Topology build timed out., retry(1/1200).
[WARNING] DISTRIBUTED(368760,ffffa8e84930,python):2025-08-28-16:19:13.410.632 [mindspore/ccsrc/distributed/cluster/cluster_context.cc:246] BuildCluster] Topology build timed out., retry(2/1200).
[WARNING] DISTRIBUTED(368760,ffffa8e84930,python):2025-08-28-16:19:13.910.728 [mindspore/ccsrc/distributed/cluster/cluster_context.cc:246] BuildCluster] Topology build timed out., retry(3/1200).
[WARNING] DISTRIBUTED(368760,ffffa8e84930,python):2025-08-28-16:19:14.410.886 [mindspore/ccsrc/distributed/cluster/cluster_context.cc:249] BuildCluster] Cluster is successfully initialized.
[WARNING] DISTRIBUTED(368760,ffffa8e84930,python):2025-08-28-16:19:14.411.077 [mindspore/ccsrc/distributed/cluster/cluster_context.cc:356] PostProcess] This node 2 rank id: 2
[WARNING] DISTRIBUTED(368760,ffffa8e84930,python):2025-08-28-16:19:20.223.670 [mindspore/ccsrc/distributed/collective/collective_manager.cc:338] CreateCommunicationGroup] Start to create communication group: hccl_world_group [const vector]{0, 1, 2, 3, 4, 5, 6, 7}, async: 0, submit_now: 1
[WARNING] DISTRIBUTED(368760,fffea8ff91e0,python):2025-08-28-16:19:20.224.565 [mindspore/ccsrc/distributed/collective/collective_manager.cc:962] CreateDeviceCommunicator] Begin initialize communication group on the device side: hccl_world_group
[WARNING] DEVICE(368760,fffe87fff1e0,python):2025-08-28-16:19:20.225.045 [mindspore/ccsrc/plugin/res_manager/ascend/collective/ascend_communication_group.cc:165] InitializeByRankTable] Start to initialize communicator by HcclCommInitClusterInfoConfig for hccl_world_group, hcclBufferSize is 200 MB. hcclDeterministic is 0
[WARNING] DEVICE(368760,fffe87fff1e0,python):2025-08-28-16:19:20.306.416 [mindspore/ccsrc/plugin/res_manager/ascend/collective/ascend_communication_group.cc:175] InitializeByRankTable] End to initialize communicator by HcclCommInitClusterInfoConfig for hccl_world_group
[WARNING] DISTRIBUTED(368760,fffea8ff91e0,python):2025-08-28-16:19:20.307.054 [mindspore/ccsrc/distributed/collective/collective_manager.cc:973] CreateDeviceCommunicator] End initialize communication group on the device side: hccl_world_group
[WARNING] ME(368760:281473515538736,MainProcess):2025-08-28-16:19:34.572.598 [mindspore/train/serialization.py:1860] For 'load_param_into_net', remove parameter prefix name: ema., continue to load.
Scanning 'coco/val2017.cache.npy' images and labels... 5000 found, 0 missing, 48 empty, 0 corrupted: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████| 5000/5000 [00:00<?, ?it/s]Scanning 'coco/val2017.cache.npy' images and labels... 5000 found, 0 missing, 48 empty, 0 corrupted: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████| 5000/5000 [00:00<?, ?it/s]
...............